bin/pc2installservice

```diff
@@ -19,74 +19,68 @@
 #
 # Author: John Buck
 
-DEBUG=
-#DEBUG="echo Would"
+DEBUG=(echo)
```

Comment:
Does this make the default value of `DEBUG `an `echo`?  The intent is to set `DEBUG `only if testing of the script is desired with no side effects.

Reply:
This was an accidental artifact from the testing process. it has been removed in 883c7bc40c1bbe99c7266efa7b509036851c7eb0